client/components/mma/upgrade/UpgradeSupportThankYou.tsx

```diff
@@ -47,11 +48,12 @@ export const UpgradeSupportThankYou = () => {
 
 	const location = useLocation();
 	const routerState = location.state as UpgradeRouterState;
-	const chosenAmount = routerState?.chosenAmount.toFixed(2);
+	const amountPayableToday = formatAmount(routerState?.amountPayableToday);
+	const chosenAmount = formatAmount(routerState?.chosenAmount);
```

Comment:
can we do the formatting within the markup like the other components? In another PR I've used this value in a `>` expression below, which works (Javascript ✨ ) but feels wrong to use a string for